reference/configuration.html.md

```diff
@@ -203,12 +203,16 @@ For each external port you want to accept connections on, you will need a `servi
   [[services.ports]]
     handlers = ["http"]
     port = "80"
+    
+    # optional
+    http_options = { compress = false }
 ```
 
 This example defines an HTTP handler on port 80.
 
 * `handlers` : An array of strings, each string selecting a handler process to terminate the connection with at the edge. Here, the ["HTTP" handler](/docs/services/#http) will accept HTTP traffic and pass it on to the internal port of the application which we defined earlier.
 * `port` : A string which selects which external ports you want Fly to accept traffic on. You can configure an application to listen for global traffic on ports 25, 53, 80, 443, 5000, 8443, 25565, and ports 10000 - 10100.
+* `http_options` : A table that specifies configuration options for HTTP handlers. It is entirely optional and can be left out.
```

Comment:
Maybe just say `An optional table to specify configuration` and cull the second sentence.